internal/cmd/api-key/command_list.go

```diff
@@ -164,17 +165,15 @@ func mapResourceIdToUserId(users []*ccloudv1.User) map[string]int32 {
 	return idMap
 }
 
-func (c *command) getEmail(resourceId string, resourceIdToUserIdMap map[string]int32, usersMap map[int32]*ccloudv1.User, serviceAccountsMap map[string]bool) string {
+func (c *command) getEmail(resourceId string, auditLog *ccloudv1.AuditLog, resourceIdToUserIdMap map[string]int32, usersMap map[int32]*ccloudv1.User, serviceAccountsMap map[string]bool) string {
 	if _, ok := serviceAccountsMap[resourceId]; ok {
 		return "<service account>"
 	}
 
 	userId := resourceIdToUserIdMap[resourceId]
 
-	if user, err := c.Client.Auth.User(); err == nil {
-		if auditLog := user.GetOrganization().GetAuditLog(); auditLog != nil && auditLog.GetServiceAccountId() == userId {
-			return "<auditlog service account>"
-		}
+	if auditLog != nil && auditLog.GetServiceAccountId() == userId {
```

Comment:
Why pass the whole audit log struct if we just need the audit log service account ID?

Reply:
Right, good point. I'll make this change tomorrow morning before I merge.